lib/plugins/aws/provider.js

```diff
@@ -962,6 +962,7 @@ class AwsProvider {
               type: 'object',
               properties: {
                 artifact: { type: 'string' },
+                patterns: { type: 'array', items: { type: 'string' } },
```

Comment:
Let's maintain alphabetical order

Reply:
They were not in alphabetical order in the service-level schema. I fixed it there.

Reply:
> I fixed it there.

Good call, thank you!